src/liballoc/borrow.rs

```diff
@@ -142,6 +142,7 @@ impl<T> ToOwned for T
 /// abs_all(&mut input);
 /// ```
 ///
+/// Another example showing how to keep `Cow` in a struct:
 /// ```
```

Comment:
Now please add an "empty" line before the example. 😆

Reply:
Done